IFC4x3/PropertyEnumerations/c/PEnum_CompactionMethod.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Does this enumeration require any documentation?

Reply:
Added in latest commit.